packages/frontend/src/components/common/SpaceActionModal/CreateSpaceModalContent.tsx

```diff
@@ -155,8 +159,16 @@ const CreateSpaceModalContent: FC<CreateSpaceModalBody> = ({
                             variant="light"
                             icon={<MantineIcon icon={IconInfoCircle} />}
                         >
-                            Nested spaces inherit permissions from their root
-                            space.
+                            This space will inherit permissions from the top
+                            level space: "
```

Comment:
should't this be parent space: "" ?

Reply:
it depends 🤷 

because if I try to create a  grandchild space, it will inherit permissions from child space which inherits permissions from parent space - so... I wonder if the messaging needs to be clearer this way?

We do the same in "Share space modal"

Reply:
gotcha! thanks!